src/Card/Card.scss

```diff
@@ -6,6 +6,12 @@
   width: 100%;
 }
 
+a .pgn__card {
+  color: $gray-700;
+  display: inline-block;
```

Comment:
nit : should this be an inline-flex vs. inline-block?

Reply:
I think in this case we can leave the block element inside the link. The obsolete HTML 4 specification forbade such nesting, but in [HTML 5 the rules](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) have changed. We can wrap a link, if necessary, in elements that do not contain interactive elements (other links) inside.
Also, in this case, it is important for us to use exactly the inline-block because text decorations do not work on "floating" elements. Inline flex unfortunately will not work.